plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -188,7 +188,8 @@ func init() {
 			// glusterfs
 			rbac.NewRule("get", "list", "watch").Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
 			rbac.NewRule("get", "create", "delete").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
-			rbac.NewRule("get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
+			// glusterfs and Azure File
+			rbac.NewRule("create", "get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
```

Comment:
@liggitt long story, at the time azure file volume was developed, there was no azure cloud provider, thus it expects storage account name and storage account key, as secrets, given in the volume source.